src/main/java/seedu/address/logic/commands/AddTaskCommand.java

```diff
@@ -0,0 +1,50 @@
+package seedu.address.logic.commands;
+import static java.util.Objects.requireNonNull;
+
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.task.Task;
+
+/**
+ * Marks a task identified by its displayed index in the GUI as completed.
+ */
+public class AddTaskCommand extends Command {
+    public static final String COMMAND_WORD = "addTask";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD
+            + ": Add a task to a taskList\n"
+            + "Parameters: Task\n"
+            + "Example: " + COMMAND_WORD + " d/<Purchase milk> D/<deadline> t/<tag>";
```

Comment:
Do you think the example can be updated to follow one format? either:
```suggestion
            + "Example: " + COMMAND_WORD + " d/Purchase milk D/2022-10-18 t/Food";
```
or:
```suggestion
            + "Format: " + COMMAND_WORD + " d/<Task details> D/<Task deadline> t/<Tag>";
```

Reply:
Updated and standardized format